certfields/README.md

```diff
@@ -18,7 +18,43 @@ Then, compile the example app:
 
 ```
 $ make
-$ ./app
+```
```

Comment:
Getting a build error with app.. Wolf built with `./configure --enable-opensslextra --enable-certreq --enable-certgen --enable-keygen CFLAGS="-DWOLFSSL_ALT_NAMES"`

```
main.c:170:25: error: use of undeclared identifier 'GEN_DNS'
        if (gn->type == GEN_DNS) {
                        ^
main.c:172:32: error: use of undeclared identifier 'GEN_IPADD'
        } else if (gn->type == GEN_IPADD) {
                               ^
2 errors generated.
```

If I add `#include <wolfssl/openssl/x509v3.h>` in `main.c` it builds fine and works fine.

David

Reply:
That's odd, were the wrong headers getting included somehow (wrong options.h)? I tried this fresh using the attached script and can't reproduce. Could you try the script and let me know if you see the error still?

```
#!/bin/sh

if [ ! -d reproduce_dgarske_review ]; then
    mkdir reproduce_dgarske_review || exit 1
fi

cd reproduce_dgarske_review || exit 1

TEST_DIR=$(pwd)
INSTALL_DIR="$TEST_DIR"/install-dir
CERTGEN_DIR="$TEST_DIR"/wolfssl-examples/certgen

if [ ! -d wolfssl ]; then
    git clone https://github.com/wolfssl/wolfssl.git
    cd wolfssl || exit 1
    ./autogen.sh

    export CFLAGS="-DWOLFSSL_ALT_NAMES"
    WOLF_OPTS="\
    --prefix=$INSTALL_DIR \
    --enable-opensslextra \
    --enable-certreq \
    --enable-certgen \
    --enable-keygen \
    "

    ./configure $WOLF_OPTS || exit 1
    make install || exit 1
    cd "$TEST_DIR" || exit 1
fi

if [ ! -d wolfssl-examples ]; then
    git clone https://github.com/wolfssl/wolfssl-examples.git
fi

cd "$CERTGEN_DIR" || exit 1

cp Makefile Makefile.bak
echo "sed \"s/^WOLF_INSTALL_DIR=.*/WOLF_INSTALL_DIR=\"$INSTALL_DIR\"/\" Makefile.bak > Makefile"
sed "s%^WOLF_INSTALL_DIR=.*%WOLF_INSTALL_DIR=${INSTALL_DIR}%" Makefile.bak > Makefile
make clean
make || exit 1
./certgen_with_altnames || exit 1

if [ -f newCert.pem ] && [ -f newCert.der ]; then
    echo "SUCCESS"
else
    echo "FAILED"
fi
```